testIntegration/tests/slasHelper.test.ts

```diff
@@ -50,9 +52,13 @@ describe("slasHelpers", () => {
       .reply(200, expectedTokenResponse);
 
     const slasClient = createSlasClient(clientConfig);
-    const tokenResponse = await slasHelpers.loginGuestUser(slasClient, {
-      redirectURI: "redirect_uri",
-    });
+    // TODO: Using 'as any' to avoid type errors betwen the enum 'code' and the string 'code'. This should be fixed in the future.
```

Comment:
I think we can add this as an AC to the refactor node SDK ticket (@W-18136408@) or we can create a new ticket for fixing all the tests